playbooks/roles/content_host/tasks/main.yml

```diff
@@ -0,0 +1,9 @@
+- name: 'Install subscription-manager'
+  yum: name=subscription-manager
+
+- name: 'Install bootstrap rpm'
+  command: 'rpm -Uvh http://{{ katello_server }}/pub/katello-ca-consumer-latest.noarch.rpm'
```

Comment:
I've always had to install sub-man first, here is what I get when I remove lines 1 and 2

```
TASK [katello_client : Install bootstrap rpm] **********************************
fatal: [client3]: FAILED! => {"changed": true, "cmd": ["rpm", "-Uvh", "http://chicken/pub/katello-ca-consumer-latest.noarch.rpm"], "delta": "0:00:00.195843", "end": "2016-09-16 14:09:22.855609", "failed": true, "rc": 1, "start": "2016-09-16 14:09:22.659766", "stderr": "error: Failed dependencies:\n\tsubscription-manager is needed by katello-ca-consumer-chicken.example.com-1.0-1.noarch", "stdout": "Retrieving http://chicken/pub/katello-ca-consumer-latest.noarch.rpm", "stdout_lines": ["Retrieving http://chicken/pub/katello-ca-consumer-latest.noarch.rpm"], "warnings": ["Consider using yum, dnf or zypper module rather than running rpm"]}

```